jobs/integr8ly/pds-general.yaml

```diff
@@ -0,0 +1,300 @@
+            description: "City or Customer (5 char min.) plus the generated hash, e.g. qebrno-5d10 [required]"
+        - string:
+            name: GH_CLIENT_ID
+            description: "GitHub AOuth Client ID, required for some of the walkthroughs. Can be left empty"
+        - string:
+            name: GH_CLIENT_SECRET
+            description: "GitHub OAuth Client Secret, required for some of the walkthroughs. Can be left empty"
+        - bool:
+            name: SELF_SIGNED_CERTS
+            default: true
+            description: "Indicates whether the cluster uses self signed certificates or not.
+This needs to be changed to 'false' for Integreatly Workshop since it uses valid certificates."
+        - string:
+            name: RECIPIENTS
+            default: integreatly-qe@redhat.com
+            description: "Whitespace- or comma-separated list of recipient addresses"
+        - string:
+            name: INSTALLATION_REPOSITORY
+            default: https://github.com/integr8ly/installation.git
+            description: "Repository of the Integreatly installer"
+        - string:
+            name: INSTALLATION_BRANCH
+            default: master
+            description: "Branch of the installer repository"
+        - string:
+            name: TEST_SUITES_REPOSITORY
+            default: https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git
+            description: "Repository of the Integreatly test suites"
+        - string:
+            name: TEST_SUITES_BRANCH
+            default: master
+            description: "Branch of a repository of Integreatly test suites"
+        - choice:
+            name: TO_DO
+            description: "It specifies what stages of the pipeline will be executed.
+'heavy' means uninstall + uninstall + install + install + tests + uninstall + install"
+            choices:
+              - heavy
+              - uninstall + uninstall
+              - install + install
+              - uninstall + install
+              - tests only
+              - install + tests
+              - uninstall + install + tests
+              - none
+    dsl: |
+        try {
+          timeout(180) {
+              node('cirhos_rhel7') {        
+                  stage('Verify input') {
+                      if (!YOURCITY) {
+                          throw new hudson.AbortException('YOURCITY parameter is required!')
+                      } // if
+                  } // stage
+                  
+                  stage('Execute "Before" Workarounds') {
+                      // Example of workaround
+                      //  - `oc login` credentials depend on whether the integreatly is installed or not)
+                      //  - alternative is to change config to run this directly on bastion node (not cirhos-rhel7), no `oc login` is requred then
+                      // oc login https://master.${YOURCITY}.openshiftworkshop.com -u openmgr-clt -p r3dh4t1!  --insecure-skip-tls-verify
```

Comment:
I know this is just comment, but the user is `opentlc-mgr`

Reply:
Grr, whatever I don't double-check is just not correct. Thanks, will fix.